find_flaky_tests.py

```diff
@@ -120,53 +121,66 @@ def render_msg_header(state: AppState) -> str:
 
 
 def print_for_slack(occurrences: List[Occurrence], state: AppState):
-    json = """
-    {
-        "channel": "@CHANNEL@",
+    occurrences_by_ann_path: Dict[str, List[Occurrence]] = {}
+    for o in occurrences:
+        occurrences_by_ann_path.setdefault(o.annotation_path, []).append(o)
+    items = [i for i in occurrences_by_ann_path.items()]
+    items.sort(key=lambda i: len(i[1]), reverse=True)  # sort by number of occurrences, highest first
+
+    limit = 12
+    items = items[:limit]
```

Comment:
The diff is a bit weird here. These lines did not change at all. The definition of the json structure and this code block has just been swapped in the code